janusgraph-core/src/main/java/org/janusgraph/util/system/ConfigurationLint.java

```diff
@@ -40,7 +41,7 @@ public static void main(String args[]) throws IOException {
             System.exit(1);
         }
 
-        log.info("Checking " + args[0]);
+        log.info("Checking configuration file");
```

Comment:
Would it make sense to still print out the value of `argv[0]` here so that it's clear from the logs which file path is being validated? Otherwise, this log line might not be useful in isolation, when the user doesn't know the full path to the file that's being checked.

Reply:
@mbrukman I updated to instead sanitize the input (and make a deep copy of the characters to clear the Fortify finding).